XamlAnimatedGif.Shared/Interfaces/ICacheHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿#region
```

Comment:
Why the `#region` ? VS already allows you to collapse the `using` statements to a single line...